docs/source/_notebooks/keras-image-classifiers.rst

```diff
@@ -0,0 +1,435 @@
+
+Explaining Keras image classifier predictions
+=============================================
+
+If we have a model that takes in an image as its input, and outputs
+class scores, i.e. probabilities that a certain object is present in the
+image, then we can use ELI5 to check what is it in the image that made
+the model predict a certain class score. We do that using a method
+called 'Grad-CAM'.
+
+We will be using images from ImageNet (for more information see
+http://image-net.org/), and classifiers from ``keras.applications``.
+
+This has been tested with Python 3.7.3, Keras 2.2.4, and Tensorflow
+1.13.1.
+
+1. Explaining the model's prediction
+------------------------------------
+
+To start out, lets load our image classifier and data.
+
+.. code:: ipython3
+
+    from PIL import Image
+    from IPython.display import display
+    
+    import logging
+    import tensorflow as tf
+    tf.get_logger().setLevel(logging.ERROR) # disable Tensorflow warnings for this tutorial
+    import warnings
+    warnings.simplefilter("ignore") # disable Keras warnings for this tutorial
+    import keras
+    from keras.applications import mobilenet_v2
+    
+    import eli5
+    
+    # let's instantiate our model
+    model = mobilenet_v2.MobileNetV2(include_top=True, weights='imagenet', classes=1000)
+    
+    # and load our sample image
+    image = 'imagenet-samples/cat_dog.jpg'  # TODO: replace this image with something more interesting
```

Comment:
> TODO: replace this image with something more interesting

I think this image is perfectly fine

Reply:
https://github.com/TeamHG-Memex/eli5/pull/315/commits/4d4556e88b07bb89a101495733fae05ef62df3ed